### PR TITLE
postgrest: 7.0.0 -> 7.0.1 

### PR DESCRIPTION
**Pull request by danielstaleiny**
- change dependency protolude to depend on 0.3.0
- change configurator-pg dependency for protolude to depend on 0.3.0 as well

###### Motivation for this change
To get latest version of Postgrest which is 7.0.1
[hackage postgrest](https://hackage.haskell.org/package/postgrest)
###### Things done
Bump version of postgrest to 7.0.1
- change dependency protolude to depend on 0.3.0
- change configurator-pg dependency for protolude to depend on 0.3.0 as well

configurator-pg needs to have same protolude, so I had to bump version for it as well. 
- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [X] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/haskell-modules/configuration-hackage2nix.yaml [hunk 1]
```diff
@@ -1720,7 +1720,7 @@ default-package-overrides:
   - postgresql-libpq ==0.9.4.2
   - postgresql-orm ==0.5.1
   - postgresql-simple ==0.6.2
-  - postgrest ==7.0.0
```
Comment (cdepillabout) [inline pkgs/development/haskell-modules/configuration-hackage2nix.yaml:1723]: These are versions from the latest stackage LTS, so we don't change them by hand.   I imagine the next version of stackage will bump `postgrest` to 7.0.1, so we can just wait for that.

Comment (danielstaleiny) [reply pkgs/development/haskell-modules/configuration-hackage2nix.yaml:1723]: Oh, I didn't know that, thank you. but the rest should be updated like I did ?

Comment (cdepillabout) [reply pkgs/development/haskell-modules/configuration-hackage2nix.yaml]: Ah, no, I think everything else in this PR should happen automatically as well, so I think you can go ahead and close this.

Check out https://github.com/NixOS/nixpkgs/pull/92461 over the next couple days.

Comment (danielstaleiny) [reply pkgs/development/haskell-modules/configuration-hackage2nix.yaml]: oh, alright. thanks for the info and sorry for wasting your time. among my first PR.

**Pull request closed without merging**